### PR TITLE
Remove sourceRepoURL from notNeededPackages.json

### PR DESCRIPTION
**Pull request by jablko**
microsoft/DefinitelyTyped-tools#142

```Shell
sed --in-place /sourceRepoURL/d notNeededPackages.json
```

### CODE REVIEW

**Review by sandersn — APPROVED** [review]
This will be ready to ship after the assocated PR ships in a release of DefinitelyTyped-tools.

**Comment by sandersn** [comment]
I just shipped [v0.0.61](https://github.com/microsoft/DefinitelyTyped-tools/releases/tag/v0.0.61); when it's done publishing, I'll re-run CI.

File: scripts/not-needed.js [hunk 1]
```diff
@@ -5,17 +5,16 @@ const path = require("path");
 
 const typingsPackageName = process.argv[2];
 const asOfVersion = process.argv[3];
-const sourceRepoURL = process.argv[4];
 const libraryName = process.argv[5] || typingsPackageName;
 
 if (process.argv.length !== 5 && process.argv.length !== 6) {
-	console.log("Usage: npm run not-needed -- typingsPackageName asOfVersion sourceRepoURL [libraryName]");
+	console.log("Usage: npm run not-needed -- typingsPackageName asOfVersion [libraryName]");
```
Comment (danger-public) [inline scripts/not-needed.js:11]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Comment (sandersn) [reply scripts/not-needed.js:11]: The tabs are pre-existing; we can get rid of them some other time.

**Pull request merged**